CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - fixed libavcodec-related memory leaks (#389)
 - fixed crash in custom levels that call `level_stats` after playing an FMV (#393, regression from 2.5)
 - fixed sounds playing after demo mode ends when game is minimized (#399)
+- fixed gun pickups disappearing in rare circumstances on save load (#406)
```

Comment:
I'll test this later tonight, but this fix should also probably go in the README too.

Reply:
Good point; fixed